### PR TITLE
Initial commit to add AccelModels notebooks from AzureMlCli repo

### PR DESCRIPTION
**Pull request by paledger**

### CODE REVIEW

File: how-to-use-azureml/deployment/accelerated-models/README.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+* [Training models](accelerated-models-training.ipynb), train one of our accelerated models on the Kaggle Cats and Dogs dataset to see how to improve accuracy on custom datasets
+
+<a name="model-classes"></a>
+## Model Classes
+As stated above, we support 5 Accelerated Models. Here's more information on their input and output tensors.
+
+**Available models and output tensors**
+
+The available models and the corresponding default classifier output tensors are below. This is the value that you would use during inferencing if you used the default classifier.
+* Resnet50, QuantizedResnet50 (output_tensors = "classifier_1/resnet_v1_50/predictions/Softmax:0")
```
Comment (rastala) [inline how-to-use-azureml/deployment/accelerated-models/README.md]: If these are snippets user is supposed to enter as code please use ```backticks```

File: how-to-use-azureml/deployment/accelerated-models/accelerated-models-training.ipynb [hunk 2]
```diff
@@ -0,0 +1,849 @@
...
+    "<a id=\"custom-weights\"></a>\n",
+    "### 6.b. Traning using Custom Weights\n",
+    "\n",
+    "Because the quantized graph defintion and the float32 graph defintion share the same node names in the graph definitions, we can initally train the weights in float32, and then reload them with the quantized operations (which take longer) to fine-tune the model.\n",
+    "\n",
+    "First we train the model with custom weights but without quantization. Training is done with native float precision (32-bit floats). We load the training data set and batch the training with 10 epochs. When the performance reaches desired level or starts decredation, we stop the training iteration and save the weights as tensorflow checkpoint files. "
+   ]
+  },
+  {
+   "cell_type": "raw",
```
Comment (rastala) [inline how-to-use-azureml/deployment/accelerated-models/accelerated-models-training.ipynb]: Please check the markdown rendering of these cells, should it be code instead?

Comment (paledger) [reply how-to-use-azureml/deployment/accelerated-models/accelerated-models-training.ipynb]: is updated

File: how-to-use-azureml/deployment/accelerated-models/README.md (hunk 1 above)
Comment (paledger) [reply how-to-use-azureml/deployment/accelerated-models/README.md]: is updated, can see at new commit https://github.com/Azure/MachineLearningNotebooks/pull/343/files#diff-e8750b7b6ee961ae966efa5c495a175d

**Pull request merged**